docs/Installing-Bouncy-Castle.md

```diff
@@ -66,3 +90,88 @@ public class SimpleTest
     }
 }
 ```
+
+## Installing Bouncy Castle on Java 11
+
+Bouncy Castle library is installed in two steps on Java 11:
+- Bouncy Castle security provider needs to be configured in `java.security` configuration file.
+- Java 11 no longer provides a library extension mechanism and thus Bouncy Castle library must be installed in the web container.
+
+### Configuring Java Security for Java 11
+
+Add following record to your `${JDK_HOME}/conf/security/java.security`:
+
+```sh
+security.provider.N=org.bouncycastle.jce.provider.BouncyCastleProvider
+```
+
+... where `N` should be replaced according to your file content. Usually, there are multiple `security.provider.X` records in the file, you should chose the next in order number as `N`, for example:
+
+```sh
+#
+# List of providers and their preference orders (see above):
+#
+security.provider.1=SUN
+security.provider.2=SunRsaSign
+security.provider.3=SunEC
+security.provider.4=SunJSSE
+security.provider.5=SunJCE
+security.provider.6=SunJGSS
+security.provider.7=SunSASL
+security.provider.8=XMLDSig
+security.provider.9=SunPCSC
+security.provider.10=JdkLDAP
+security.provider.11=JdkSASL
+security.provider.12=Apple
+security.provider.13=SunPKCS11
+security.provider.14=org.bouncycastle.jce.provider.BouncyCastleProvider
+```
+
+_Warning: Configuring Bouncy Castle as the first provider (security.provider.1) may cause JVM errors._
+
+### Installing Bouncy Castle - Tomcat on Java 8
```

Comment:
... this is the line that actually confused me in the previous comment - I think it should read "... on Java 11"?